translator/config/schema.json

```diff
@@ -618,6 +618,90 @@
               },
               "additionalProperties": true
             },
+            "application_signals": {
```

Comment:
What if customer configures both `app_signals` and `application_signals` in the configuration? I don't see validation.

Reply:
application_signals should take preference. I'll add a new unit test

Reply:
Added unit test